test/SparqlAntlrParserTest.cpp

```diff
@@ -874,28 +873,22 @@ TEST(SparqlParser, SelectQuery) {
       m::GraphPattern(m::Triples({{"?x", "?y", "?z"}}));
   expectSelectQuery(
       "SELECT * WHERE { ?a <bar> ?foo }",
-      testing::AllOf(
-          m::SelectQuery(m::AsteriskSelect(), m::GraphPattern(m::Triples(
-                                                  {{"?a", "<bar>", "?foo"}}))),
-          m::pq::OriginalString("SELECT * WHERE { ?a <bar> ?foo }")));
-  expectSelectQuery(
-      "SELECT * WHERE { ?x ?y ?z }",
-      testing::AllOf(
-          m::SelectQuery(m::AsteriskSelect(), DummyGraphPatternMatcher),
-          m::pq::OriginalString("SELECT * WHERE { ?x ?y ?z }")));
-  expectSelectQuery(
-      "SELECT ?x WHERE { ?x ?y ?z } GROUP BY ?x",
-      testing::AllOf(
-          m::SelectQuery(m::VariablesSelect({"?x"}), DummyGraphPatternMatcher),
-          m::pq::OriginalString("SELECT ?x WHERE { ?x ?y ?z } GROUP BY ?x"),
-          m::pq::GroupKeys({Variable{"?x"}})));
+      testing::AllOf(m::SelectQuery(
+          m::AsteriskSelect(),
+          m::GraphPattern(m::Triples({{"?a", "<bar>", "?foo"}})))));
+  expectSelectQuery("SELECT * WHERE { ?x ?y ?z }",
+                    testing::AllOf(m::SelectQuery(m::AsteriskSelect(),
+                                                  DummyGraphPatternMatcher)));
+  expectSelectQuery("SELECT ?x WHERE { ?x ?y ?z } GROUP BY ?x",
+                    testing::AllOf(m::SelectQuery(m::VariablesSelect({"?x"}),
+                                                  DummyGraphPatternMatcher),
+                                   m::pq::GroupKeys({Variable{"?x"}})));
```

Comment:
`ParsedQuery::_originalString` is now only set at the layer of `Query` (one rule above `SelectQuery`).

Reply:
Then there probably should be a (small) test case for the `Query` rule.